src/IB/IBFESurfaceMethod.cpp

```diff
@@ -1669,7 +1668,8 @@ IBFESurfaceMethod::imposeJumpConditions(const int f_data_idx,
                         const libMesh::Point x = r + intersections[k].first * q;
                         const libMesh::Point& xi = intersections[k].second;
                         SideIndex<NDIM> i_s(i_c, axis, 0);
-                        i_s(axis) = boost::math::iround((x(axis) - x_lower[axis]) / dx[axis]) + patch_lower[axis];
+                        i_s(axis) = int(std::round((x(axis) - x_lower[axis]) / dx[axis]))
```

Comment:
`static_cast<int>`? And, if it needs a cast anyway, should we just omit the call to `std::round` and let the cast do the rounding for us?